tests/src/test/scala/services/KafkaConnectorTests.scala

```diff
@@ -148,7 +148,7 @@ class KafkaConnectorTests
       kafkaHosts.indices.foreach { i =>
         val message = createMessage()
         val kafkaHost = kafkaHosts(i).split(":")(0)
-        val startLog = s", started"
+        val startLog = s"\\[KafkaServer id=$i\\] started"
```

Comment:
This might allow you to avoid escape chars.

```suggestion
        val startLog = s"""[KafkaServer id=$i] started"""
```

Reply:
made change

Reply:
reverted.   need the \[ to avoid regex interpreting it as a character class.